iac-extensions/arm/src/test/resources/checks/IpRestrictedAdminAccessCheck/Microsoft.Network_networkInterfaces/test.bicep

```diff
@@ -0,0 +1,98 @@
+// Noncompliant@+1 {{Restrict IP addresses authorized to access administration services.}}
+resource raise_an_issue 'Microsoft.Network/networkInterfaces@2022-11-01' = {
+//                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  name: 'Raise an issue'
+  properties: {
+    ipConfigurations: [
+      {
+        properties: {
+          subnet: {
+            properties: {
+              networkSecurityGroup: {
+                properties: {
+                  securityRules: [
+                    {
+                      properties: {
+                        direction: 'Inbound'
+                        //         ^^^^^^^^^< {{Sensitive direction}}
+                        access: 'Allow'
+                        //      ^^^^^^^< {{Sensitive access}}
+                        protocol: 'TCP'
+                        //        ^^^^^< {{Sensitive protocol}}
+                        destinationPortRange: '*'
+                        //                    ^^^< {{Sensitive destination port range}}
+                        sourceAddressPrefix: '*'
+                        //                   ^^^< {{Sensitive source address prefix}}
+                      }
+                    }
+                  ]
+                }
+              }
+            }
+          }
+        }
+      }
+    ]
+  }
+}
+
+resource microsoft_Network_networkInterfaces_Compliant 'Microsoft.Network/networkInterfaces@2022-11-01' = {
```

Comment:
Please change all resource names in all files to lower camel case, as suggested in Bicep Best practises: https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/best-practices#names 

Some day it can be code smell rule: https://sonarsource.atlassian.net/browse/SONARIAC-1062

Reply:
For the reference: I used this command to change names `find . -type f -exec sed -i.bak -E 's/_([a-z0-9])/\U\1/g' {} \;`